Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/12.0/InspectorBackendCommands.js

```diff
@@ -96,7 +85,7 @@ InspectorBackend.activateDomain("Canvas", ["page"]);
 
 // Console
 InspectorBackend.registerDomain("Console", ["itml", "javascript", "page", "service-worker", "worker"]);
-InspectorBackend.registerEnum("Console.ChannelSource", {XML: "xml", JavaScript: "javascript", Network: "network", ConsoleAPI: "console-api", Storage: "storage", Appcache: "appcache", Rendering: "rendering", CSS: "css", Security: "security", ContentBlocker: "content-blocker", Media: "media", WebRTC: "webrtc", Other: "other"});
```

Comment:
this should also probably be removed from `Source/JavaScriptCore/inspector/protocol/Console.json`